Base/QTGUI/qSlicerStyle.cxx

```diff
@@ -193,15 +194,16 @@ QRect qSlicerStyle::subControlRect(ComplexControl control, const QStyleOptionCom
 QPalette qSlicerStyle::standardPalette()const
 {
   QPalette palette = this->Superclass::standardPalette();
-  palette.setColor(QPalette::Button, "#fcfcfc");
-  palette.setColor(QPalette::Light, "#c8c8c8");
-  palette.setColor(QPalette::Midlight, "#e6e6e6");
-  palette.setColor(QPalette::Dark, "#aaaaaa");
-  palette.setColor(QPalette::Mid, "#c8c8c8");
-  palette.setColor(QPalette::Base, Qt::white);
-  palette.setColor(QPalette::Window, Qt::white);
-  palette.setColor(QPalette::Shadow, "#5a5a5a");
-  palette.setColor(QPalette::AlternateBase, QColor("#e4e4fe"));
+  // TODO: Use standardPalette as defined in qSlicerLightStyle
+
+  #ifdef Q_OS_WIN
+    // Qt on macOS already dynamically picks light/dark theme based on the OS setting
+    QSettings settings("HKEY_CURRENT_USER\\Software\\Microsoft\\Windows\\CurrentVersion\\Themes\\Personalize", QSettings::NativeFormat);
+    if (settings.value("AppsUseLightTheme") == 0)
+      {
+      // TODO: Use standardPalette as defined in qSlicerDarkStyle
```

Comment:
@lassoan Can you provide guidance, or commit the change yourself, to use the standardPalette() option from qSlicerLightStyle and qSlicerDarkStyle here in qSlicerStyle? I'm trying not to duplicate the individual palette set options from those classes here.

Reply:
Since qSlicerDarkStyle depends on qSlicerStyle, the dark palette has to be either in qSlicerStyle or in a third class that both classes can access. Since this situation is temporary (the palette will move into qSlicerDarkStyle when Qt fully supports Windows and Linux dark mode), I think it is simpler not to add the palette to a third class, but create a `QPalette qSlicerStyle::standardDarkPalette()const` method and use that in both qSlicerStyle and qSlicerDarkStyle.

Reply:
Ok, I will try that!